ssg/templates.py

```diff
@@ -213,8 +213,12 @@ def build_rule(self, rule_id, rule_title, template, langs_to_generate):
         local_env_yaml["rule_title"] = rule_title
         local_env_yaml["products"] = self.env_yaml["product"]
         for lang in langs_to_generate:
-            self.build_lang(
-                rule_id, template_name, template_vars, lang, local_env_yaml)
+            try:
+                self.build_lang(
+                    rule_id, template_name, template_vars, lang, local_env_yaml)
+            except Exception as e:
+                print("Error building rule {0}".format(rule_id), file=sys.stderr)
```

Comment:
Could the language matter?
Depending on the template vars can be specific for a language.

Suggestion for message:
```suggestion
                print("Error building templated {0} content for rule {1}".format(lang, rule_id), file=sys.stderr)
```

Reply:
Ahh good idea, thanks @yuumasato!